govc/vm/create.go

```diff
@@ -109,6 +110,7 @@ func (cmd *create) Register(ctx context.Context, f *flag.FlagSet) {
 	f.BoolVar(&cmd.force, "force", false, "Create VM if vmx already exists")
 	f.StringVar(&cmd.controller, "disk.controller", "scsi", "Disk controller type")
 	f.StringVar(&cmd.annotation, "annotation", "", "VM description")
+	f.BoolVar(&cmd.firmwareEfi, "firmware-efi", false, "Use firmware type EFI (otherwise BIOS)")
```

Comment:
I'd prefer to use the same type as the field as we do elsewhere for enums.  Even though there's only 2 options in this case, a bool is tempting to use.
```go
f.StringVar(&cmd.firmware, "firmware", "", "Firmware type [bios|efi]")
```
Some existing enum flag examples:
```go
govc/datastore/create.go:132:	f.Var(&cmd.Type, "type", fmt.Sprintf("Datastore type (%s)", strings.Join(allTypes, "|")))
govc/device/scsi/add.go:53:		fmt.Sprintf("SCSI controller type (%s)", strings.Join(ctypes, "|")))
govc/device/usb/add.go:48:		fmt.Sprintf("USB controller type (%s)", strings.Join(ctypes, "|")))
govc/dvs/portgroup/spec.go:41:		fmt.Sprintf("Portgroup type (%s)", strings.Join(ptypes, "|")))
```